src/blueprints/Do.Blueprints.Service.Application/Domain/Model/TypeModelMembers.cs

```diff
@@ -79,14 +79,16 @@ ConstructorModel BuildConstructor(ConstructorInfo constructorInfo)
                 );
             }
 
-            PropertyModel BuildProperty(PropertyInfo property) =>
-                new(
+            PropertyModel BuildProperty(PropertyInfo property)
+            {
+                return new(
```

Comment:
these doesn't cause a problem in `dotnet build`, undo them, and configure your local dev env with respect to `.editorconfig`